### ISSUE TITLE
[BUG] request() got an unexpected keyword argument

### ISSUE THREAD

**Issue by comato** [schemathesis/schemathesis#1734]
After update to  schemathesis-3.19.0 this code is not working
```
PARAMS    = { "from": "2022-01-01T00:00:00", "to": "2022-03-01T00:00:00", "top": 1}

def test_api(case):
    case.call_and_validate(headers=HEADERS, params=PARAMS)


        try:
            with self.operation.schema.ratelimit():
>               response = session.request(**data)  # type: ignore
E               TypeError: request() got an unexpected keyword argument 'top'

/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/schemathesis/models.py:367: TypeError
```

I changed part of code on line 354 in schemathesis/models.py
```
        if params is not None:
            #_merge_dict_to(data, "params", params)
            data['params'] = params # <<<<<<<<<
```

After this change it is working again.

Is it bug? Or I using incorrect call of case.call_and_validate ?

**Comment by Stranger6667** [comment]
Hi! It is indeed a bug, thank you for reporting!
Will fix it soon

**Comment by Stranger6667** [comment]
@comato It should be fixed in `3.19.1` which was released a few minutes ago.
